### PR TITLE
Moving kops validation tests out of travis

### PR DESCRIPTION
**Pull request by chrislovecnm**
/assign @krzyzacy @BenTheElder

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -11192,6 +11192,51 @@
...
+      "./hack/verify-gofmt.sh"
+    ],
+    "scenario": "execute",
+    "sigOwners": [
+      "sig-lifecycle"
+    ]
+  },
+  "pull-kops-verify-govet": {
+    "args": [
+      "make govet"
```
Comment (chrislovecnm) [inline jobs/config.json]: Can I do this here?  Not certain

Comment (krzyzacy) [inline jobs/config.json]: the args need to be separated, do "make", "govet"

**Comment by krzyzacy** [comment]
update prow/config.yaml and testgrid/config.yaml as well :-)

File: jobs/config.json (hunk 1 above)
Comment (chrislovecnm) [reply jobs/config.json]: 👍

**Comment by chrislovecnm** [comment]
Thanks @krzyzacy still learning all of these hoops I have to jump through :)

**Comment by krzyzacy** [comment]
follow https://github.com/kubernetes/test-infra#create-a-new-job I guess

File: prow/config.yaml [hunk 2]
```diff
@@ -719,6 +719,102 @@ presubmits:
...
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--clean"
+
+  - name: pull-kops-verify-packages
+    agent: kubernetes
+    context: pull-kops-verify-govet
```
Comment (chrislovecnm) [inline prow/config.yaml]: typo

**Comment by chrislovecnm** [comment]
/hold

working on cleaning up some bugs :)

**Pull request merged**